applications/luci-app-packet-capture/htdocs/luci-static/resources/view/packet_capture/tcpdump.js

```diff
@@ -0,0 +1,293 @@
+'use strict';
+'require rpc';
+'require uci';
+'require ui';
+'require fs';
+'require form';
+'require network';
+'require tools.widgets as widgets';
+
+var eventSource,
+	captureFilePoll,
+	hostName;
+
+function stopTcpdump() {
+	fs.exec("/usr/sbin/packet_capture_stop").then(function(replay) {
+		if(eventSource) { 
```

Comment:
Please put a space between the keyword (`if`) and the left paren (`if (eventSource)`). Also please omit the curly braces for single-statement loops or conditionals:

```
if (eventSource)
    eventSource.close();
```

Reply:
Ok.